src/components/header/header.scss

```diff
@@ -1,17 +1,25 @@
 .header-nav {
-  text-align: center;
-  padding-top: 20px;
   position: fixed;
-  left: 35vw;
+  padding-top: 20px;
+  left: calc( 50% - 90px); 
```

Comment:
The logo is still not centered for me. There is no need to do the left positioning and calculation. Just use plain centering or flexbox to center the logo.

.header-nav can be removed here as well as in the html so you can just add the styles to the header tag.